src/com/Dryft/models/Car.java

```diff
@@ -0,0 +1,25 @@
+package com.Dryft.models;
+
+public class Car {
+    private final String name;
+    private final int wheels;
+    private final int capacity;
```

Comment:
Also include a field car type which will include values like SUV,Micro ,Mini etc

Reply:
Also add costPerKm for that car